packaging/macos/sign_notarize_staple.sh

```diff
@@ -1,58 +1,58 @@
 #!/bin/bash
 
+set -e
```

Comment:
Implementation note: This fixes the issue that we ran notarization, even if signing the DMG didn't succeed (which in that case is bound to fail).